EIPS/eip-4955.md

```diff
@@ -75,6 +81,69 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 There are many projects which need custom properties in order to display a current NFT. Each project may have its own way to render the NFTs and therefore they need different values. An example of this is the metaverses like Decentraland or TheSandbox where they need different 3d models to render the NFT based on the visual/engine of each. NFTs projects like Cryptopunks, Bored Apes, etc. can create the 3d models needed for each project and therefore be supported out of the box.
 
+The main differences between the projects that are rendering 3d NFTs (models) are:
+
+- **Armatures**
+For example, every metaverse uses its own armature. There is [a standard](https://cascadeur.com/help/getting_started/import_fbxdae/using_mixamo_rig_in_cascadeur) for humanoids but it is not being used for every metaverse and not all the metaverses use humanoids. For example, Decentraland has a different esthetic than Cryptovoxels and TheSandbox. It means that every metaverse will need a different model and they may have the same extension (GLB, GLTF)
+
+![Screen Shot 2022-04-06 at 14.24.42|690x355](https://ethereum-magicians.org/uploads/default/original/2X/4/4c0c9d1fda4579e97225499d5d1a9c1fd5fda962.jpeg)
```

Comment:
You'll need to move these images into `assets/eip-4955/` and link to them there.